app/controllers/state_file/questions/primary_state_id_controller.rb

```diff
@@ -1,6 +1,6 @@
 module StateFile
   module Questions
-    class AzPrimaryStateIdController < QuestionsController
+    class PrimaryStateIdController < QuestionsController
```

Comment:
[pebble-ish, i guess?] given this is a shared place now, is it better to have differentiating logic in the controller instead of the view? For example, we could do

```ruby
class PrimaryStateIdController < QuestionsController
  include ReturnToReviewConcern

  def edit
    super
    render "state_file/questions/#{current_state_code}_primary_state_id/_#{current_state_code}_primary" 
  end
end
```

i think that would remove the need for the views that just have if/elsif blocks, which i much prefer (both fewer view files and fewer conditionals). But I am not sure if it adds complication in the controller/form flow. I don't _think_ it does, but I'm not the expert on the controller magic.